sdk/cognitiveservices/cognitiveservices-luis-authoring/README.md

```diff
@@ -15,82 +15,89 @@ npm install @azure/cognitiveservices-luis-authoring
+#### nodejs - Authentication, client creation and listApplicationVersionPatternFeatures features as an example written in TypeScript.
 
-##### Install @azure/ms-rest-nodeauth
+##### Install @azure/ms-rest-azure-js
 
 ```bash
-npm install @azure/ms-rest-nodeauth
+npm install @azure/ms-rest-azure-js
 ```
 
 ##### Sample code
 
 ```typescript
-import * as msRest from "@azure/ms-rest-js";
-import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
-import { LUISAuthoringClient, LUISAuthoringModels, LUISAuthoringMappers } from "@azure/cognitiveservices-luis-authoring";
-const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
-
-msRestNodeAuth.interactiveLogin().then((creds) => {
-  const client = new LUISAuthoringClient(creds, subscriptionId);
-  const appId = ec7b1657-199d-4d8a-bbb2-89a11a42e02a;
-  const versionId = "testversionId";
-  const skip = 1;
-  const take = 1;
-  client.features.listPhraseLists(appId, versionId, skip, take).then((result) => {
+import { CognitiveServicesCredentials } from "@azure/ms-rest-azure-js";
+import { LUISAuthoringClient } from "@azure/cognitiveservices-luis-authoring";
+
+let authoringKey = process.env["luis-authoring-key"];
+const creds = new CognitiveServicesCredentials(authoringKey);
+
+// check the following link to find your region
+// https://docs.microsoft.com/en-us/azure/cognitive-services/luis/luis-reference-regions
+const region = "<your-region>";
+const client = new LUISAuthoringClient(
+  creds,
+  "https://" + region + ".api.cognitive.microsoft.com/"
+);
+
+const appId = "<your-app-id>"; // replace this with your appId.
+const versionId = "0.1"; // replace with version of your luis application. Initial value will be 0.1
+
+const skip = 1;
+const take = 1;
+
+client.features
+  .listApplicationVersionPatternFeatures(appId, versionId, { skip, take })
+  .then((result) => {
     console.log("The result is:");
     console.log(result);
+  })
+  .catch((err) => {
+    console.error(err);
   });
-}).catch((err) => {
-  console.error(err);
-});
 ```
 
-#### browser - Authentication, client creation and listPhraseLists features as an example written in JavaScript.
+#### browser - Authentication, client creation and listApplicationVersionPatternFeatures features as an example written in JavaScript.
 
-##### Install @azure/ms-rest-browserauth
+##### Install @azure/ms-rest-azure-js
 
 ```bash
-npm install @azure/ms-rest-browserauth
+npm install @azure/ms-rest-azure-js
 ```
 
 ##### Sample code
 
-See https://github.com/Azure/ms-rest-browserauth to learn how to authenticate to Azure in the browser.
-
 - index.html
+
 ```html
 <!DOCTYPE html>
 <html lang="en">
   <head>
     <title>@azure/cognitiveservices-luis-authoring sample</title>
-    <script src="node_modules/@azure/ms-rest-js/dist/msRest.browser.js"></script>
-    <script src="node_modules/@azure/ms-rest-browserauth/dist/msAuth.js"></script>
+    <script src="node_modules/@azure/ms-rest-azure-js/dist/cognitiveServicesCredentials.js"></script>
```

Comment:
```suggestion
```

The `@azure/ms-rest-azure-js` package does not have a browser bundle and I have logged https://github.com/Azure/ms-rest-azure-js/issues/126 to track this issue.

Until that issue is fixed, we can use the `ApiKeyCredentials` instead